ArcGISRuntimeSDKQt_CppSamples/EditData/EditWithBranchVersioning/EditWithBranchVersioning.cpp

```diff
@@ -50,7 +50,7 @@ namespace
 EditWithBranchVersioning::EditWithBranchVersioning(QObject* parent /* = nullptr */):
   QObject(parent),
   m_map(new Map(Basemap::streetsVector(this), this)),
-  m_cred(new Credential("editor01", "editor01.password", this))
+  m_cred(new Credential("username", "password", this))
```

Comment:
As I recall, we needed a default sample username/password in the sample so we could perform the edits - is that correct? If so, and we are approved to share the credentials, it is fine to hardcode it like this. If a user is expected to supply their own username/password, we should use AuthenticationView to prompt for a login

Reply:
@tanneryould I am going to take a jump and guess that you intended to put the credentials in the space for `username` and `password`. 

Otherwise initializing the credential in the initialization list is fine. Not sure if that's what you are asking in this scenario.

Reply:
@jared-2016  

I intended that the sample's user would overwrite the placeholder `username` and `password` with their own credentials. To be more clear, my question was asking how we normally write out placeholder/example credentials.

@ldanzinger I'm not sure if we're approved to share the credentials for this. I'll check, but perhaps AuthenticationView would be the better way to go then.